code/game/objects/structures/lattice.dm

```diff
@@ -19,8 +19,10 @@
 /obj/structure/lattice/Initialize(mapload)
 	. = ..()
 	for(var/obj/structure/lattice/LAT in loc)
-		if(LAT != src)
-			QDEL_IN(LAT, 0)
+		if(LAT == src)
+			continue
+		stack_trace("multiple lattices found in ([loc.x], [loc.y], [loc.z])")
+		return INITIALIZE_HINT_QDEL
```

Comment:
Hmmmmm :thinking:

Reply:
Lattices arent meant to stack so this prevents that